renovate.json

```diff
@@ -0,0 +1,30 @@
+{
+  "$schema": "https://docs.renovatebot.com/renovate-schema.json",
+  "extends": [
+    "config:recommended",
+    ":semanticCommits",
+    ":semanticCommitTypeAll(chore)"
+  ],
+  "labels": [
+    "dependencies"
+  ],
+  "enabledManagers": [
+    "custom.regex"
+  ],
+  "customManagers": [
+    {
+      "customType": "regex",
+      "fileMatch": [
+        "^GO_VERSION$"
+      ],
+      "depTypeTemplate": "language",
+      "matchStrings": [
+        "^(?<currentValue>.+)\\n"
+      ],
+      "depNameTemplate": "https://github.com/golang/go",
+      "autoReplaceStringTemplate": "{{{newValue}}}",
+      "datasourceTemplate": "golang-version"
```

Comment:
I think it would be better to depend on the actual Go docker images https://hub.docker.com/_/golang as the docker image releases can lag a little behind the Go release.

Reply:
Thanks that's true. I was worried about how to make renovate get the tags right

Seems there is already a solution , will put it in later 

https://docs.renovatebot.com/modules/versioning/docker/